### PR TITLE
Add test classes

### PR DESCRIPTION
**Pull request by Ko-Khan**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/articlecommands/FilterArticleCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    }
+    //EP Only Status
+    @Test
+    public void execute_onlyStatus_showFilteredList() throws ParseException {
+        ArticleMatchesStatusPredicate predicate = new ArticleMatchesStatusPredicate("DRAFT");
+        expectedModel.updateFilteredArticleList(predicate);
+        assertCommandSuccess(new FilterArticleCommand("DRAFT", EMPTY_TAG, EMPTY_START, EMPTY_END),
+                model, FilterArticleCommand.MESSAGE_SUCCESS, expectedModel);
+    }
+    //ep Only tag
```
Comment (Howlong11) [inline src/test/java/seedu/address/logic/commands/articlecommands/FilterArticleCommandTest.java:48]: just minor error, but ep should be EP..?

File: src/test/java/seedu/address/logic/commands/articlecommands/FilterArticleCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    @Test
+    public void execute_onlyTag_showFilteredList() throws ParseException {
+        Tag tag = new Tag("Science");
+        ArticleMatchesTagPredicate predicate = new ArticleMatchesTagPredicate(tag);
+        expectedModel.updateFilteredArticleList(predicate);
+        assertCommandSuccess(new FilterArticleCommand("", tag, EMPTY_START, EMPTY_END),
+                model, FilterArticleCommand.MESSAGE_SUCCESS, expectedModel);
+    }
+    //ep Only date
```
Comment (Howlong11) [inline src/test/java/seedu/address/logic/commands/articlecommands/FilterArticleCommandTest.java:57]: same here as well

**Review by Howlong11 — APPROVED** [review]
looks good, the minor issues can be ignored I think

File: src/test/java/seedu/address/logic/commands/articlecommands/FilterArticleCommandTest.java (hunk 1 above)
Comment (Ko-Khan) [reply src/test/java/seedu/address/logic/commands/articlecommands/FilterArticleCommandTest.java:48]: Thanks for spotting, but i accidentally merged into this branch, so I can't really change now

**Pull request merged**